controllers/applicationvolumereplication_controller.go

```diff
@@ -1066,6 +1121,85 @@ func (a *AVRInstance) updateAVRStatus() error {
 	return nil
 }
 
+/*
+Description: create a new ManagedClusterView object, or update the existing one with the same name.
+Requires:
+	1) meta: specifies MangedClusterView name and managed cluster search information
+	2) viewscope: once the managed cluster is found, use this information to find the resource.
+		Optional params: Namespace, Resource, Group, Version, Kind. Resource can be used by itself, Kind requires Version
+Returns: ManagedClusterView, error
+*/
+func (r *ApplicationVolumeReplicationReconciler) getOrCreateManagedClusterView(
```

Comment:
(nit) We are following the convention of called functions below callers, so this would go below `getManagedClusterResource`